lib/utils/cli-flags.js

```diff
@@ -182,27 +192,31 @@ module.exports = {
         {
             name: 'target',
             alias: 't',
+            usage: '--target <build target>',
             defaultValue: 'web',
             type: String,
             group: ADVANCED_GROUP,
             description: 'Sets the build target',
         },
         {
             name: 'watch',
+            usage: '--watch',
             type: Boolean,
             alias: 'w',
             group: BASIC_GROUP,
             description: 'Watch for files changes',
         },
         {
             name: 'hot',
+            usage: '--hot',
             alias: 'h',
             type: Boolean,
             group: ADVANCED_GROUP,
             description: 'Enables Hot Module Replacement',
         },
         {
             name: 'sourcemap',
+            usage: '--sourcemap <sourcemap>',
```

Comment:
Maybe we should show the alternatives provided by webpack:
`--sourcemap <sourcemap|eval|>` etc.

Basically what we have here: https://webpack.js.org/configuration/devtool/#devtool